### PR TITLE
Update gem dependencies

### PR DESCRIPTION
**Pull request by andrewjamesbrown**
This commit simply updates the dependencies for more recent versions.

### CODE REVIEW

**Comment by andrewjamesbrown** [comment]
Hi @marcparadise, is there anything I can do to help move this along?  Thanks!

File: Gemfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ gemspec
 
 gem 'rest-client', :github => 'opscode/rest-client'
 
-gem 'oc-chef-pedant', :github => 'opscode/oc-chef-pedant', :tag => '2.0.0'
+gem 'oc-chef-pedant', :github => 'chef/oc-chef-pedant', :tag => '2.0.0'
 
-gem 'chef', :github => 'opscode/chef', :tag => '12.1.0.rc.0'
+gem 'chef', :github => 'chef/chef', :tag => '12.4.0'
```
Comment (tyler-ball) [inline Gemfile]: Can you update this to 12.4.1 now?  12.4.0 had some regressions.

**Comment by tyler-ball** [comment]
:+1:

File: Gemfile (hunk 1 above)
Comment (andrewjamesbrown) [inline Gemfile]: Done.

**Comment by tyler-ball** [comment]
Hey @andrewjamesbrown - thanks for this PR.  I'm going to merge it into the WIP branch and we can continue discussion in https://github.com/chef/chef-zero/pull/117

**Pull request merged**